lib/pavilion/result_parsers/json.py

```diff
@@ -64,8 +64,7 @@ def parse_json(self, file, stop_at):
                 return json.load(file)
             except json.JSONDecodeError as err:
                 raise ValueError(
-                    "'{}' is invalid JSON"
-                    .format(err)
+                    "Invalid JSON: {}".format(err)
```

Comment:
This change will cause some unit tests to fail, since they rely on a hardcoded error string for a `self.assertTrue(string.endswith())` assertion, which is a pattern I've seen in other unit tests. Is there an assertion I should use here that would be more resilient to changes like this?